addons/account/static/tests/reconciliation_tests.js

```diff
@@ -1620,5 +1622,135 @@ QUnit.module('account', {
+                            assert.deepEqual(
+                                _.pick(args.args[0][0].new_mv_line_dicts[idx],
+                                       'account_id', 'name', 'credit', 'debit', 'journal_id'),
+                                {account_id: 287, name: "dummy text", credit: 0, debit: 180, journal_id: 8},
+                                "2nd reconciliation: The gift line is correct"
+                            );
+                            assert.deepEqual(
+                                _.pick(args.args[0][0].new_mv_line_dicts[1 - idx],
+                                       'account_id', 'name', 'credit', 'debit', 'tax_line_id'),
+                                {account_id: 287, name: "Tax 20.00%", credit: 0, debit: 36, tax_line_id: 6},
+                                "2nd reconciliation: The tax line is correct"
+                            );
+                            break
+                    }
+                }
+                return this._super.apply(this, arguments);
+            }
+        });
+
+        var $body = $('body');
+        $body.addClass('debug');
+        clientAction.appendTo($body);
+
+        // First account is already in the wanted state to test the
+        // reconciliation. We reconcile it now.
+        reconcileNo = 1;
+        clientAction.$("button.o_reconcile.btn.btn-primary:first").click();
+
+        // The second account isn't in a the wanted state, the next
+        // lines (until we click on the reconciliation button) changes
+        // the state to make it reconciable.
+
+        var $top = clientAction.$('.o_reconciliation_line:first');
+        var $table_to_reconcile = $top.find('.match');
+
+        // Select invoice of 1k$, payment of 1k$ and payment of 180$
+        _.each([6, 19, 21], function(id) {
+            $table_to_reconcile.find('tr.mv_line[data-line-id=' + id + ']:first td:first-child').click();
+        });
+        
+
+        // Store the money in excess to the "account receivable" account with 20% taxes
+        $top.find("table tfoot tr td:first").click();
+        var $reconcile_form = $top.find(".create");
+        $reconcile_form.find('.create_account_id input').click();
+        $('.ui-autocomplete .ui-menu-item a:contains(101200 Account Receivable)').trigger('mouseover').trigger('click');
+        $reconcile_form.find('.create_tax_id input').click();
+        $('.ui-autocomplete .ui-menu-item a:contains(Tax 20.00%)').click();
+        $reconcile_form.find('.create_journal_id input').click();
+        $('.ui-autocomplete .ui-menu-item a:contains(company 1 journal)').click();
+        $reconcile_form.find('.create_label input').val('dummy text').trigger('input');
+        
+        // Verify the two (gift + tax) lines were added to the list
+        var $newLines = $top.find('tr.mv_line[data-line-id^=createLine]');
```

Comment:
I think you can select the right section by using

        var widget = clientAction.widgets[0];